redaxo/src/addons/mediapool/lib/service_media.php

```diff
@@ -5,6 +5,12 @@
  */
 final class rex_media_service
 {
+    public const orderby = [
```

Comment:
args, konstanten-namen immer in großbuchstaben ;) 

```suggestion
    public const ORDER_BY = [
```